src/pages/errors/no-service-worker.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * Page to display a user friendly message when `navigator.serviceWorker` is not available.
+ */
+
+import React from 'react'
+import About from '../../components/About.jsx'
+import Header from '../../components/Header.jsx'
+import '../default-page-styles.css'
+
+export default function NoServiceWorkerErrorPage (): React.JSX.Element {
+  return (
+    <>
+      <Header />
+      <main className='pa4-l bg-red-muted mw7 mb5 center pa4'>
+        <h1>Service Worker Error</h1>
+        <p>
+          This page requires a service worker to be available. Please enable
+          service workers in your browser and try again.
+        </p>
```

Comment:
Fun fact: Firefox does not expose SW API in private windows ;-)

Let's improve message to save time for Firefox users trying this in private window, wondering why it does not work. 

```suggestion
        <p>
          This page requires a service worker to be available. Please ensure that
          your browser supports service workers and that they are enabled
          (navigator.serviceWorker is present).
        </p>
        <p>
          If you are using Firefox, please note that service workers are disabled
          in private browsing mode. Please try again in a regular browsing window.
        </p>
```

Reply:
This is a great improvement. There's currently no way to enable service workers in private ff, but it could be useful to link to https://bugzilla.mozilla.org/show_bug.cgi?id=1320796